test_schemas/en/test_repeating_sections_with_hub_and_spoke.json

```diff
@@ -501,7 +501,10 @@
         {
             "id": "personal-details-section",
             "title": "Personal Details",
-            "summary": { "show_on_completion": true },
+            "summary": {
+                "page_title": "Summary for person {list_item_index}",
```

Comment:
Why has this been added here rather than in `test_custom_page_titles.json`?

Reply:
Cheating :P